src/routes/v1/userInfo.route.ts

```diff
@@ -14,7 +14,9 @@ const userInfoRoute = (app: Express) => {
 		userInfoController.updateUserInfo(req, res);
 	});
 
-
+	router.patch("/alpaca/:userId", (req, res) => {
+		userInfoController.updateAlpacaToken(req,res);
+	})
```

Comment:
Missing `async/await` here

```suggestion
	router.patch("/alpaca/:userId", async (req, res) => {
		await userInfoController.updateAlpacaToken(req,res);
	})
```

Reply:
i don't think its necessary since the function is async